src/core/config.ts

```diff
@@ -19,6 +19,9 @@ export enum BITCOIN_NETWORK {
   MAINNET = 'MAINNET',
 }
 
+export const KEEPERWEBSITE = 'https://help.bitcoinkeeper.app/';
+export const MAINKEEPERWEBSITE = 'https://bitcoinkeeper.app/'
```

Comment:
change KEEPERWEBSITE to KEEPER_KNOWLEDGEBASE
change MAINKEEPERWEBSITE to KEEPER_WEBSITE_BASE_URL

Reply:
Changed.